src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -205,6 +247,26 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
         return this.memberships[0];
     }
 
+    public onCallEncryption = (event: MatrixEvent): void => {
+        const userId = event.getSender();
+        const content = event.getContent<EncryptionKeyEventContent>();
+        const encryptionKey = content["m.encryption_key"];
+        const deviceId = content["m.device_id"];
+
+        if (
+            !userId ||
+            !deviceId ||
+            !encryptionKey ||
+            typeof deviceId !== "string" ||
+            typeof encryptionKey !== "string"
+        ) {
+            throw new Error("Malformed m.call.encryption_key");
```

Comment:
I added the user ID to the equivalents of this because I wasn't finding the error messages terribly helpful, would be good to do the same here.